ifrs17-template/Test/ReimportWithDifferentScope.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
This file name is inconsistent with all other files in this folder. Appending "Test" at the end would make it consistent.